listen.py

```diff
@@ -43,6 +43,41 @@ def from_json(cls, j):
             data=j.get('track_metadata')
         )
 
+    @classmethod
+    def from_influx(cls, row):
+        """ Factory to make Listen objects from an influx row
+        """
+        dt = datetime.strptime(row['time'] , "%Y-%m-%dT%H:%M:%SZ")
+        t = int(dt.strftime('%s'))
+        mbids = []
+        if row.get('artist_mbids'):
+            for mbid in row.get('artist_mbids').split(","):
```

Comment:
This is something that you copied, but let's really try and be consistent with quotes here. I don't mind which one we use, let's just not have two different ones on the same line! (unless we need to use one type to include the other type)